### PR TITLE
add doc to run share-now in cli

### PR DESCRIPTION
**Pull request by xzf0587**
1. add doc  to run share-now in cli.

### CODE REVIEW

File: share-now/cli.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    )
+    GO
+    CREATE TABLE [UserVoteEntity](
+	    [VoteID] [int] PRIMARY KEY IDENTITY,
+	    [PostID] [int] NOT NULL,
+	    [UserID] [uniqueidentifier] NOT NULL,
+    )
+    GO
+    ```
+1. Refer [add user](https://github.com/OfficeDev/TeamsFx/blob/dev/docs/fx-core/sql-help.md#step-2-add-database-user-manually) to add user in database.
```
Comment (zhyuzha) [inline share-now/cli.md:43]: Refer to [manually add user] ...

File: share-now/cli.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+1. Once deployment and SQL script are completed, you can preview the app running in Azure. In Visual Studio Code, open `Run and Debug` and select `Launch Remote (Edge)` or `Launch Remote (Chrome)` in the dropdown list and Press `F5` or green arrow button to open a browser.
```
Comment (zhyuzha) [inline share-now/cli.md:44]: Once you have successfully created DB Table and added user, you can now try previewing the app running in Azure.  ...

File: share-now/README.md [hunk 3]
```diff
@@ -20,6 +20,7 @@ Share Now promotes the exchange of information between colleagues by enabling us
 - How to connect to Azure SQL DB and how to do CRUD operations in DB.
 
 ## Try the Sample
+>Here is the steps to try the sample **in Visual Studio Code**. If you want to try it in command line, please refer to [Command Line](cli.md)
```
Comment (zhyuzha) [inline share-now/README.md:23]: Here are the instructins to run the sample in **Visual Studio Code**. You can also try to run the app using TeamsFx CLI tool, refer to [Try sample with TeamsFx CLI](cli.md)

Comment (xzf0587) [reply share-now/README.md]: It is fixed. Thanks.

File: share-now/cli.md (hunk 2 above)
Comment (xzf0587) [reply share-now/cli.md]: It is fixed. Thanks.

File: share-now/cli.md (hunk 1 above)
Comment (xzf0587) [reply share-now/cli.md]: It is fixed. Thanks.

**Review by zyxiaoyuer — APPROVED** [review]

File: share-now/cli.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+## Try the Sample in Command Line
```
Comment (jiayi-ruan) [inline share-now/cli.md:1]: should the title be "Try sample with TeamsFx CLI"? to be consistent with the Readme.

File: share-now/cli.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+1. Install [Node.js](https://nodejs.org/en/download/) (use the latest v14 LTS release)
+1. To install the TeamsFx CLI, use the npm package manager:
+    ```
+    npm install -g @microsoft/teamsfx-cli
+    ```
+1. create share-now project.
```
Comment (jiayi-ruan) [inline share-now/cli.md:7]: Capitalize the first letter of the first word in the sentence?

File: share-now/cli.md (hunk 4 above)
Comment (xzf0587) [reply share-now/cli.md]: It is fixed. Thanks.

File: share-now/cli.md (hunk 5 above)
Comment (xzf0587) [reply share-now/cli.md]: It is fixed. Thanks.

**Pull request merged**